packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1047,6 +1082,52 @@ class DataTable extends StatelessWidget {
   }
 }
 
+/// Height settings for DataRows.
+/// Either a fixed row height or a row height that adjusts to its content can be specified.
+@immutable
+class DataTableRowHeight with Diagnosticable {
+  /// Creates height settings where all rows are created with the same fixed height.
+  /// [fixedHeight] defaults to kMinInteractiveDimension to adhere to the Material Design specifications.
+  const DataTableRowHeight.fixed({
+    double? height = kMinInteractiveDimension
```

Comment:
make this a required parameter and put the default value in DataTable or DataTableTheme class.

If developers ever want to use this method, they are not satisfy with the `kMinInteractiveDimension`, so there is no point to default it to the `kMinInteractiveDimension`